src/GitHub.App/ViewModels/PullRequestDetailViewModel.cs

```diff
@@ -478,11 +488,13 @@ IObservable<Unit> DoCheckout(object unused)
 
         IObservable<Unit> DoPull(object unused)
         {
+            usageTracker.IncrementPullRequestPullCount(IsFromFork).Forget();
             return pullRequestsService.Pull(repository);
```

Comment:
Do we want to count this all the time, regardless of whether it succeeds? Do we want to track failures?